src/Components/Panels/Toolbox/PassabilityControls.module.scss

```diff
@@ -0,0 +1,23 @@
+.wrapper {
+  border: 1px solid rgba(#fff, 0.3);
+  display: flex;
+  flex-direction: column;
+  padding: 1rem 1.25rem;
+  margin: auto 0;
+}
+
+.legend {
+  font-size: 0.85rem;
+  font-weight: bold;
+  width: auto;
```

Comment:
```suggestion
```

Let the default vault of `width: 100%` from the Bootstrap Reboot stylesheet take effect here